src/cairoWriter/writers/cairoFunctionDefinitionWriter.ts

```diff
@@ -122,7 +122,13 @@ export class CairoFunctionDefinitionWriter extends CairoASTNodeWriter {
 
   private getReturns(node: CairoFunctionDefinition, writer: ASTWriter): string {
     if (node.kind === FunctionKind.Constructor) return '';
-    return `-> (${writer.write(node.vReturnParameters)})`;
+
+    const returnStr = writer.write(node.vReturnParameters);
+    const paramLen = node.vReturnParameters.vParameters.length;
+    // Cairo1 does not need to always return a tuple as former versions
+    if (paramLen > 1) return `-> (${returnStr})`;
```

Comment:
Cairo 1 does not need to return a tuple, then why parentheses around return arguments?

Reply:
If you want to return more than one value then the tuple is needed, the writer for params return a string of the form 'p1:t1, p2:t2, ..., pN:tN' so that string is wrapped with the parenthesis.

Reply:
Oh, you're right, I thought it was `paramLen < 1`. I guess it should be done on AST level though, so a tuple node containing multiple typenames I guess. It might require more work though so let's merge the PR and we'll fix it later